### ISSUE TITLE
Need to support require.resolve()

### ISSUE THREAD

**Issue by jurassix** [jurassix/refactoring-codemods#5]
Add support for `require.resolve()` since these can be relative paths.

**Comment by ahutchings** [comment]
Might be able to knock https://github.com/jurassix/atom-javascript-refactor/issues/24 out with this, too.

**Comment by jurassix** [comment]
Yeah I'm seeing the same thing with __.css__. `create-react-app` using css file imports from JS, we should preserve refs when the css files move.

**Comment by jurassix** [comment]
Also I'm seeing imports of __.svg__ files too. These should preserve when moving them.

**Comment by jurassix** [comment]
Because of the following possible whitelist, I'm just going to open this up 💃  to anything.

1. .json
1. .css
1. .(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga
1. .jsx
1. .react.js

**Comment by jurassix** [comment]
Update. I'm down to a single failing scenario. I should resolve it soon.
